csharp/src/Apache.Arrow.Flight/FlightInfo.cs

```diff
@@ -1,4 +1,4 @@
-﻿// Licensed to the Apache Software Foundation (ASF) under one or more
+// Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
Can you please edit your PR so there's no diff in this area of the file?

Reply:
It appears to be removing the "UTF-8 BOM". I don't know what the state of all the other source files is in this regard; I know that it's generally not recommended to use one (and I expect it's not required to compile C#) but I know there are definitely things in the Microsoft world that work better when there is one.

On the whole, if we think we should get rid of these, we should probably do it for all the files in one change.

Reply:
Yeah, I wasn't aware this (and probably others) had BOMs. +1 to make any change in a dedicated PR.